src/main/resources/scripts/xslt/transformToMods.xsl

```diff
@@ -2023,6 +2028,7 @@
         <xsl:if test="field[@name='Udgivelsesland'] or field[@name='Country'] or
           field[@name='Udgivelsessted'] or
           field[@name='Location of origin'] or
+          field[@name='Location of sender'] or
```

Comment:
If element "location of sender" should be removed

Reply:
Tak, den smuttede i mergen. Jeg fjerner.